src/main/java/net/snowflake/client/core/SFSessionProperty.java

```diff
@@ -85,7 +85,16 @@ public enum SFSessionProperty {
   DISABLE_GCS_DEFAULT_CREDENTIALS("disableGcsDefaultCredentials", false, Boolean.class),
 
   JDBC_ARROW_TREAT_DECIMAL_AS_INT("JDBC_ARROW_TREAT_DECIMAL_AS_INT", false, Boolean.class),
-  DISABLE_SAML_URL_CHECK("disableSamlURLCheck", false, Boolean.class);
+
+  DISABLE_SAML_URL_CHECK("disableSamlURLCheck", false, Boolean.class),
+
+  // Used to determine whether a hardcoded value should be used for the formatter (for backwards
```

Comment:
Hardcoded value of what? I didn't follow this issue so I'm similar to any customer who doesn't know the topic at all. For me this description is not comprehensible.